draft-thomson-webpush-protocol.xml

```diff
@@ -269,9 +273,10 @@
           are defined:
           <list style="hanging">
             <t hangText="push service:">
-              This resource is used in <xref
-              target="subscribe">Subscribing</xref>.  It is configured into user
-              agents.
+              This resource is used to subscribe <xref
+              target="message_subscription">to messages</xref> and 
+              <xref target="receipt_subscription">receipts</xref>.
+              It is configured into user agents and application servers.
```

Comment:
This looks related to your comment below about the source/creation of the resource for subscribing to receipts.